src/annotator/anchoring/test/types-test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var types = require('../types');
+
+var TextQuoteAnchor = types.TextQuoteAnchor;
+var TextPositionAnchor = types.TextPositionAnchor;
+
+// These are primarily basic API tests for the anchoring classes. Tests for
+// anchoring a variety of HTML and PDF content exist in `html-test` and
+// `pdf-test`.
+describe('Anchoring classes', function () {
```

Comment:
Agreed. I'll change the name here to match the module. I think the closest we have to a convention is that the top-level describe matches:

1. The name of the 'default' export if the file has one (ie. if it does `module.exports = someFunction` then the describe block is 'someFunction')
2. Otherwise the name is the module filename